detect-new-secrets.sh

```diff
@@ -49,5 +59,17 @@ if [ "$(cat $new_secrets_file)" = "[]" ]; then
     echo "No secrets found"
     exit 0
 fi
-markdown_from_new_secrets | tee "$GITHUB_STEP_SUMMARY"
+
+markdown_limited_advice=$(markdown_from_new_secrets)
+markdown_console_advice=$(advice_if_none_are_secret_short)
+markdown_jobs_summary_advice=$(advice_if_none_are_secret_verbose)
+
+# Print a short message to the console
+echo "$markdown_limited_advice"
+echo "$markdown_console_advice"
+
+# Write a more detailed message to the jobs summary
+echo "$markdown_limited_advice" > "$GITHUB_STEP_SUMMARY"
+echo "$markdown_jobs_summary_advice" >> "$GITHUB_STEP_SUMMARY"
```

Comment:
I am using 4 `echo`'s instead of 2 `printf` because I don't want there to be any changes to the `\`'s in `$markdown_jobs_summary_advice`, and I want the console + jobs summary printing code to look the same